package.json

```diff
@@ -23,7 +23,7 @@
     "version:latest": "lerna version --exact --force-publish --no-private --create-release github --conventional-commits --yes",
     "release:latest": "lerna publish from-package --dist-tag latest --yes",
     "release:next": "lerna publish from-package --canary --preid next --pre-dist-tag next --yes",
-    "build:docs": "cd docs && ./vercel-build.sh",
+    "build:docs": "cd website && yarn install && yarn build",
```

Comment:
Do we need this command when it is not used in the GitHub workflow nor it is described in the `CONTRIBUTING.md` file?

Reply:
It's used by Vercel during deployment as a [Build command](https://vercel.com/docs/concepts/deployments/configure-a-build#build-command)

Reply:
I guess we can just set `yarn install && yarn build` for the Vercel build command, will check that

Reply:
Probably it would be better to leave this here. In case of any changes, it will be enough to change the command here and not change it on the Vercel side. What do you think?

Reply:
Sure, I am okay with that 🙂